tests/core/selection/fake.js

```diff
@@ -1096,5 +1096,36 @@ bender.test( {
 
 			assert.areEqual( '<p>[[placeholder]]</p>', editor.getData() );
 		} );
+	},
+
+	// (#898)
+	'test image long alt visible in editor': function() {
+		var bot = this.editorBot,
+			editor = bot.editor;
+
+		bot.setData( '<p>[<span id="bar">bar</span>]</p>', function() {
+			var altContainer, styles, expected, item;
+			editor.getSelection().fake( editor.document.getById( 'bar' ), '<i>foo</i>' );
+
+			altContainer = editor.editable().findOne( '[data-cke-hidden-sel]' );
+
+			// On IE and Edge < 14 element should have `display:none`.
```

Comment:
Not essentially true, this code path is for IE only. In fact, this comment is useless thus it explain code, not business reasons.